frontend/src/metabase/App.jsx

```diff
@@ -44,7 +44,10 @@ const getErrorComponent = ({ status, data, context }) => {
   }
 };
 
-const PATHS_WITHOUT_NAVBAR = [/\/dataset\/.*\/query/];
+const PATHS_WITHOUT_NAVBAR = [
+  /\/dataset\/.*\/query/,
+  /\/dataset\/.*\/metadata/,
```

Comment:
@kulyk should this now be updated?
```suggestion
  /\/model\/.*\/query/,
  /\/model\/.*\/metadata/,
```